proto/viam/app/v1/app.proto

```diff
@@ -508,8 +525,49 @@ message Fragment {
   string organization_owner = 4 [(tagger.v1.tags) = "bson:\"organization_owner\" json:\"owner\""];
   bool public = 5 [(tagger.v1.tags) = "bson:\"public\" json:\"public\""];
   google.protobuf.Timestamp created_on = 6 [(tagger.v1.tags) = "bson:\"created_on\""];
+  string organization_name = 7;
+  bool readonly = 8;
```

Comment:
For the getFragment request, we return this field, even though it's not currently in the proto definition. It is currently always set to false, but adding this in here just to mimic current behavior.